lib/include/openamp/rpmsg.h

```diff
@@ -123,7 +134,7 @@ struct rpmsg_device {
  * Returns number of bytes it has sent or negative error value on failure.
  */
 int rpmsg_send_offchannel_raw(struct rpmsg_endpoint *ept, uint32_t src,
-			      uint32_t dst, const void *data, int size,
+			      uint32_t dst, const void *data, int len,
```

Comment:
No need to change the input arg name.

Reply:
> No need to change the input arg name.

I make this change for consistency, because all other send functions(e.g. rpmsg_send) use len. Does it make sense?

Reply:
i suggest to put this a separate commit

Reply:
Ok, I will split this change to another patch